Tests/ChainsafeRPCTests.cs

```diff
@@ -129,51 +132,66 @@ public void GetGasPriceTest()
         [Test]
         public void GetTransactionReceiptTest()
         {
-            var latestBlockWithTx = _infuraProvider.GetBlockWithTransactions().Result;
+            var latestBlockWithTx = _ganacheProvider.GetBlockWithTransactions().Result;
             var firstTransactionInBlock = latestBlockWithTx.Transactions[0];
             var receipt =
-                _infuraProvider.GetTransactionReceipt(
+                _ganacheProvider.GetTransactionReceipt(
                     firstTransactionInBlock.Hash).Result;
             Assert.AreEqual(firstTransactionInBlock.Hash, receipt.TransactionHash);
             Assert.AreEqual(firstTransactionInBlock.BlockHash, receipt.BlockHash);
         }
 
+        /**
+         * To run this test you need to deploy a contract using your Ganache instance and the contract needs to have a safeMint function that increments the tokenId counter
+         * Example contract .sol file:
+         * contract Contract is ERC721, Ownable {
+                using Counters for Counters.Counter;
+                Counters.Counter private _tokenIdCounter;
+                constructor() ERC721("Contract", "MNFT") {}
+
+                function safeMint(address to) public onlyOwner {
+                    uint256 tokenId = _tokenIdCounter.current();
+                    _tokenIdCounter.increment();
+                    _safeMint(to, tokenId);
+                }
+            }
+         */
         [Test]
         public void CallContractMethodTest()
```

Comment:
this one was complicated, it has a lot of manual preconditions because of using ganache:

- Initiate Ganache instance locally
- Deploy a contract using Remix (needs to have the safeMint function)
- Replace _contractAddress with your contract address deployed